client/src/data/languages/en.js

```diff
@@ -228,7 +228,11 @@ export default {
     'DESCRIPTION_ADD_CUSTOM_TAG': 'here you can add a new custom tag',
     'DESCRIPTION_EDIT_CUSTOM_TAG': 'here you can edit a custom tag',
     'CUSTOM_FIELDS': 'Custom fields',
-
+    'PERMISSIONS': 'Permissions',
+    'TICKET_CREATION_PERMISSION': 'Ticket creation permission',
+    'TICKET_COMMENT_PERMISSION': 'Ticket comment permission',
+    'TICKET_NUMBER_RETURN_PERMISSION' : 'Ticket number return permission',
```

Comment:
This message doesn't seem enough to specify what the checkbox does, and I can't think of another better string. What do you think about adding a tooltip to specify what this option is doing?